packages/bootstrap/scss/chip/_variables.scss

```diff
@@ -70,8 +69,8 @@ $kendo-chip-lg-line-height: $kendo-chip-line-height !default;
 
 /// The calculated height of the Chip.
 /// @group chip
-$kendo-chip-calc-size: calc( #{$kendo-chip-line-height * 1em} + calc( #{$kendo-chip-padding-y} * 2 ) + #{$kendo-chip-border-width * 2} ) !default;
-$kendo-chip-sm-calc-size: calc( #{$kendo-chip-sm-line-height * 1em} + calc( #{$kendo-chip-sm-padding-y} * 2 ) + #{$kendo-chip-border-width * 2} ) !default;
+$kendo-chip-calc-size: calc( #{$kendo-chip-line-height} * 1em + calc( #{$kendo-chip-padding-y} * 2 ) + #{$kendo-chip-border-width} * 2 ) !default;
```

Comment:
I don't think we need the nested calc here. In the first and third part of the equation, we are not using calcs: #{$kendo-chip-line-height} * 1em and #{$kendo-chip-border-width} * 2, only in the second part: calc( #{$kendo-chip-padding-y} * 2 )